conda_env.yaml

```diff
@@ -12,3 +12,5 @@ dependencies:
 - scipy=1.6.0
 - matplotlib=3.3.4
 - pyyaml=5.4.1
+- python-kaleido=0.2.1
```

Comment:
what is python-kaleido for?

Reply:
Plotly uses kaleido backend for exporting static images, as described [here](https://plotly.com/python/static-image-export/). It has been added in advance to support exporting bar plots from plotly, rather than matplotlib.